backend/src/main/kotlin/cz/loono/backend/api/service/PushNotificationService.kt

```diff
@@ -0,0 +1,116 @@
+package cz.loono.backend.api.service
+
+import com.google.gson.Gson
+import cz.loono.backend.api.dto.BadgeTypeDto
+import cz.loono.backend.api.dto.ExaminationTypeDto
+import cz.loono.backend.api.dto.SexDto
+import cz.loono.backend.db.model.Account
+import cz.loono.backend.notification.NotificationDefinition
+import cz.loono.backend.notification.NotificationResponse
+import cz.loono.backend.notification.PushNotification
+import okhttp3.Call
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+class PushNotificationService {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    init {
+        if (ONESIGNAL_API_KEY.isEmpty()) {
+            logger.warn("ONESIGNAL_API_KEY ENV variable is not set.")
```

Comment:
Časom by to chcelo nejaký monitoring. Možno spraviť sondu/actuator?

Reply:
Actuator uz je out of the box, staci ho nakonfigurovat, pridavam jako git issue https://github.com/cesko-digital/loono-be/issues/75

Reply:
Zalezi co checes @martin-ladecky monitorovat? Actuator uz na serveru mame.

Reply:
Chcem mať istotu, že si všimneme, že premenná nie je nastavená. To, že to bude v logu je nám na houby :). Ideálne email a/alebo warning v Kibana/AWS kontrola.

Reply:
Chces to resit v ramci tohodle PR? :-) Za me je to na minimalne celou story na monitoring a alerty. Protoze to neni jen tahle jedna promenna.

Reply:
Ok, poriešime obecne inde. Súhlas.

Reply:
Take souhlasim s @rposkocil je to docela komplexni problem ten monitoring. Meli bychom zavest jak Kibanu tak i Grafanu

Jinak actuator uz mame, bezi a je otevreny ramci unauthenticated endpointu, takze zaviram https://github.com/cesko-digital/loono-be/issues/75